fluidos_model_orchestrator/common.py

```diff
@@ -21,11 +20,11 @@ class Resource:
     id: str
     cpu: str = None
     memory: str = None
-    architecture: Optional[str] = None
-    gpu: Optional[str] = None
-    ephemeral_storage: Optional[str] = None
-    persistent_storage: Optional[str] = None
-    region: Optional[str] = None
+    architecture: str | None = None
+    gpu: str | None = None
```

Comment:
Hi Stefano, 

just out of curiosity, I've never seen this syntax `str | None` I'm assuming it's better than Optional[str] but was wondering what was the rational for the change?

Reply:
I thought it looks better. I've no strong evidence that a notation is better than the other. Actually, there would be also `architecture: Union[str, None]` as additional option.